src/world/Explosion.php

```diff
@@ -214,11 +250,35 @@ public function explodeB() : bool{
 				}
 				$this->world->setBlockAt($pos->x, $pos->y, $pos->z, $airBlock);
```

Comment:
You can just do `$this->world->setBlockAt($pos->x, $pos->y, $pos->z,
				 	isset($this->fireIgnitions[$hash]) &&
					$block->getSide(Facing::DOWN)->getSupportType(Facing::UP) === SupportType::FULL ? $fireBlock : $airBlock
				);` here instead of adding another foreach